testdata/instrumenter/bco/SelectorExpr.go

```diff
@@ -0,0 +1,13 @@
+package instrumenter
+
+// https://go.dev/ref/spec#Selectors
+
+// TODO: Add systematic tests.
+
+// selectorExpr covers the instrumentation of [ast.SelectorExpr], which has
+// the branch.
```

Comment:
I don't know what you wanted to express by changing
> which has the expression field X.

into
> which has the branch.

What value does this changed comment have?